introduction.md

```diff
@@ -1,69 +1,66 @@
-Lastly, the fourth  part digs into remote computing, bash scripting and slurm.
-
-### C++ lessons
-
-1. What is a program
-1. Compilation and debugging/debuggers
+- Project Configuration
+- Version Control with Git, including remote server visualization with GitHub
+
+The standard for high performing scientific computing is C++.
+Therefore, the primary focus of the following sections will be to provide guidance for the C++ programming language. 
+Examples in this document will serve to as a template for configuring and setting up more complicated projects. 
+Let it be noted that the following content is subjective and not intended to be comprehensive.
+It reflects personal preferences and conclusions regarding the C++ language.
+That is, C++ is a a very large and general purpose language, allowing for freedom of expression and emphasis. 
+To this extent, little importance is placed on concepts such as inheritance, highlighting its relevance simply for applications such as "Curiously Recurring Template Parameters". 
+Furthermore, this document will address modern C++, encompassing language constructs from 2011 and onward, and a review of the standard library.
+
+## Chapters and Content
+
+The content will be divided into 4 parts:
+- C++ jargon, syntax, an idioms
+- Structure and documentation of multi source projects
+- Example projects and design consideration
+- Command Line, Remote Computing, bash scripting, Slurm, and Git
+
+### C++
+
+1. What is a program?
+1. Compilation and debugging
 1. Variables, types, values and templates
-1. Functions, operators, lambdas (anonymous functions), function overloading, and variadic templates
+1. Functions, operators, lambdas, function overloading, and variadic templates
 1. User-defined types (classes)
 1. Pointers, arrays, references, and smart-pointers
-1. Memory management and memory owndership models
-1. Digression: imperative programming vs object-oriented programming vs functional programming
+1. Memory management and owndership models
+1. Digression/Paradigms: Imperative vs Object-Oriented vs Functional Programming
 1. Move/copy construction/assignment
-1. Our best friend, std::optional, and error handling
-1. More on the standard template library (STL
+1. std::optional and error handling
+1. More on the standard template library (STL)
 
-What we will not cover is:
+ Topics briefly considered are:
 
 1. Inheritance
 1. Template metaprogamming
-1. Common data structures and algorithms (such as queues and sort), this is because these are provided by the standard template library
+1. Common data structures and algorithms, these are provided by the standard template library
 1. Concurrency computation models
 1. Disributed computation models
 
-### Meat and bones of a project
+### Project Configuration
 
 1. Setting up a multi-source file project
 1. Building projects with shell scripts
 1. Building projects with Makefiles
```

Comment:
I may replace this with Meson, since CMake comes with Ninja and Makefiles, so they can be discussed (briefly) in there